src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25747.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue25747 : _IssuesUITest
+{
+	public Issue25747(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Updating a Border.TranslationY does not work";
+
+	[Test]
+	[Category(UITestCategories.Border)]
+	[Category(UITestCategories.InputTransparent)]
+	public void TranslationYShouldWork()
+	{
+		App.WaitForElement("Button");
+		App.Click("Button");
+		App.Click("Button");
+
+		VerifyScreenshot();
```

Comment:
Snapshots available in the latest build. Could you commit them?

![image](https://github.com/user-attachments/assets/c73cdd27-ade1-467b-9c13-275243abbf35)

Reply:
Right! Thanks for letting me know. Added